### PR TITLE
Add functionality to create new food drives

### PR DESCRIPTION
**Pull request by estchang**
# Issue #360 

## Description

* Add functionality to create new food drives

## Change Type (delete non-relevant options)

- 💡 New feature (non-breaking change which adds functionality)

## Dependencies

- #361 should be merged before this one

### CODE REVIEW

**Review by QahtanAlJammali — COMMENTED** [review]
Looks pretty solid, though I think Flake8 is failing on Travis, might need some fixing or flake8 bypass if needed with #noqa 
We can approve right away once those and merge conflicts are solved.
Thank you so much for putting these together so fast. I think the modify/delete will have to go onto sprint 12 since our last pull requests are due tonight!

**Comment by QahtanAlJammali** [comment]
Since we are not going below 90% I suggest we merge by tonight, and tests can be added in spring 12.

**Review by arsalananwar11 — APPROVED** [review]
LGTM, thanks Estella!

**Pull request merged**
